api/src/main/java/com/jmanday/api/login/LoginService.kt

```diff
@@ -0,0 +1,14 @@
+package com.jmanday.api.login
+
+import retrofit2.http.Body
+import retrofit2.http.Headers
+import retrofit2.http.POST
+
+interface LoginService {
+    @Headers(
+        "x-api-key: Aj8vfl8S5QTEkhUD"
+    )
+    @POST("login")
+    fun login(@Body loginRequest: LoginRequest): LoginResponseDto
```

Comment:
This will make it a blocking request which can possibly block your app's UI thread when being executed on it. I'd highly recommend to avoid that, since nothing will force the caller to execute this in a separate thread. Instead, I'd use coroutines. Have you used them before?
```
@POST("login")
suspend fun login(@Body request: LoginRequest): LoginResponseDto
```
A suspend function is forced to be called in a coroutine scope or inside another suspend function, so it's asynchronous by default. Normally, you'd make the call to this in a repository:
```
class AuthRepositoryImpl(private val api: LoginService): AuthRepository {

    override fun login(email: String, password: String): Result<User, NetworkError> {
        return try {
            api.login(LoginRequest(...))
        } catch(e: HttpException) {
            // parse e.code() into right error type
        }
    }
}
```
This can the be safely called from your ViewModel:
```
viewModelScope.launch {
    repository.login(...)
}
```